packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -30,16 +30,24 @@ import {
     PaneResizeDirection,
     Row
 } from "@microsoft/fast-layouts-react";
+import { isRTL } from "@microsoft/fast-application-utilities";
 
 export enum ComponentViewSlot {
     example = "canvas-example-view",
     detailExample = "canvas-detail-view-example",
     detailDocumentation = "canvas-detail-view-documentation"
 }
 
+export enum Direction {
+    rtl = "rtl",
+    ltr = "ltr"
+}
+
 export interface ISiteProps {
     title: string;
     formChildOptions: IFormChildOption[];
+    onUpdateLTR?: (ltr: Direction) => void;
```

Comment:
yea lets change this - you're updating document direction, not "ltr"

Reply:
good point, fixing